tests/stages/test_discovery.py

```diff
@@ -20,7 +20,8 @@ def test_should_find_invalidated_test_dependencies(self):
         assert len(
             find_invalidated_projects_for_stage(projects, Stage.BUILD, [History(0, "revision", touched_files)])) == 1
         assert len(
-            find_invalidated_projects_for_stage(projects, Stage.TEST, [History(0, "revision", touched_files)])) == 2
+            find_invalidated_projects_for_stage(projects, Stage.TEST, [
+                History(0, "revision", touched_files)])) == 2  # TODO: the tests say it's one, not sure what's changed
```

Comment:
This is equal to 1 now, was like that before my changes as well. Not sure what changed here